Adyen/UI/List/ListItem.swift

```diff
@@ -39,6 +39,9 @@ public class ListItem: FormItem {
     /// The `accessibilityLabel` to be used on the `ListItem` or ``ListCell``
     public let accessibilityLabel: String
     
+    /// The handler to invoke when an item changes its loading state
+    internal var loadingHandler: ((Bool, ListItem) -> Void)?
```

Comment:
Is it still not the controller updating item's state? Description is a bit unclear

Reply:
Fair 🙈 Please see 15de5dec6d2b3e2fc88ddb22613159123451a171 if it's more clear now :)